cmd/partitiontest_linter/analyzer/testdata/analyzer_testdata_test.go

```diff
@@ -0,0 +1,65 @@
+package analyzer_testdata
+
+import (
+	"testing"
+
+	"../../../../test/partitiontest"
```

Comment:
github.com/algorand/go-algorand/test/partitiontest

Reply:
this fails for me:
```
Running tool: /usr/local/go/bin/go test -timeout 30s -coverprofile=/var/folders/m7/v0208hbj5p7643101fcjkpl00000gp/T/vscode-go78tJNa/go-code-cover github.com/algorand/go-algorand/cmd/partitiontest_linter/analyzer

analyzer_testdata_test.go:7:2: cannot find package "github.com/algorand/go-algorand/test/partitiontest" in any of:
	/usr/local/go/src/github.com/algorand/go-algorand/test/partitiontest (from $GOROOT)
	/Users/jack/projects/algojack.go-algorand/cmd/partitiontest_linter/analyzer/testdata/src/github.com/algorand/go-algorand/test/partitiontest (from $GOPATH)
/Users/jack/projects/algojack.go-algorand/cmd/partitiontest_linter/analyzer/testdata/analyzer_testdata_test.go:7:2: could not import github.com/algorand/go-algorand/test/partitiontest (invalid package name: "")
```

Reply:
updated with a comment, as this is input file.